sip-ri/src/main/java/gov/nist/javax/sip/header/extensions/TargetDialogList.java

```diff
@@ -0,0 +1,38 @@
+package gov.nist.javax.sip.header.extensions;
+
+import java.text.ParseException;
+import java.util.Iterator;
+
+import gov.nist.javax.sip.header.*;
+
+/**
+ * This interface represents the Target-Dialog SIP header list, as defined by RFC 4538.
+ * @author ValeriiaMukha
+ */
+public final class TargetDialogList extends SIPHeaderList<TargetDialog> {
```

Comment:
Target Dialog is not a list. only one such header can exist in a given SIP Message I believe. See https://www.tech-invite.com/fo-abnf/tinv-fo-abnf-sip-h-targetdlg.html

Reply:
deleted it